Addressing issue #58 with guidance for handling non-DDR queries for resolver.arpa (#61)
ietf-wg-add/draft-ietf-add-ddr: draft-ietf-add-ddr.md

configured to present the appropriate TLS certificate when no SNI is present
for both DoT and DoH.

## Handling non-DDR queries for resolver.arpa

DNS resolvers that support DDR by responding to queries for _dns.resolver.arpa

> SHOULD treat all domain names under resolver.arpa as locally served per {{?RFC6303}}.

Reviewer: ```suggestion
SHOULD treat resolver.arpa as a locally served zone per {{!RFC6303}}.
```
